dataland-frontend/src/components/resources/frameworkDataSearch/TwoLayerDataTable.vue

```diff
@@ -117,7 +117,12 @@
 
       <Column field="subcategoryKey"></Column>
       <template #groupheader="slotProps">
-        <span :data-test="slotProps.data.subcategoryKey" :id="slotProps.data.subcategoryKey" style="cursor: pointer">
+        <span
+          :data-test="slotProps.data.subcategoryKey"
+          :data-row-header-click="slotProps.data.subcategoryKey"
```

Comment:
Value doesn't really matter. Should be static

Reply:
Please do that here similar to in `SfdrPanel`. I liked the way you did that there.

Reply:
You mean something like: data-test: twoLayerDataTable?

Reply:
This is done. Change to use the attribute without  value.. `data-row-header-click`.

Reply:
👍